py-polars/docs/source/reference/series/index.rst

```diff
@@ -19,8 +19,8 @@ This page gives an overview of all public Series methods.
    export
    list
    modify_select
```

Comment:
`modify_select` appears in the docs as `Manipulation/selection`, so is correctly before `miscellaneous`